### ISSUE TITLE
Cannot assign a device for operation

### ISSUE THREAD

**Issue by tdf1995** [tensorflow/models#4968]
when i tried to train the resnet_v2 in the flowers datasets using train_image_classifier.py，i got this error:

InvalidArgumentError (see above for traceback): Cannot assign a device for operation 'gradients/softmax_cross_entropy_loss/xentropy_grad/LogSoftmax': Could not satisfy explicit device specification '/device:GPU:0' because no supported kernel for GPU devices is available.

NVIDIA GeForce GTX 1060 5GB
CUDA9.0
CUDNN v7
I found some related solutions: set the clone_on_CPU to True,but it's too slow,and I need to use GPU to train,how can i solve this problem

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
